src/stats.rs

```diff
@@ -32,7 +31,7 @@ pub struct Stats {
     pub tunnel_connections: usize,
 
     /// Messages sent by us on different routes.
-    routes: [usize; MIN_GROUP_SIZE],
+    routes: Vec<usize>,
```

Comment:
But I think for `Stats` we shouldn't even put it in: We should just extend the vector whenever we need to. That would simplify the `Stats`' API and be one less point where we need to make changes later.